modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices_{context}"]
+= Adding kernel arguments to enable IOMMU for the CPU
+
+To enable IOMMU extension for the BIOS associated with a PCI device, add kernel arguments to Intel or AMD devices. Create the `MachineConfig` object file to add the kernel arguments.
+
+.Prerequisites
+. Administrative privilege to a working OpenShift Container Platform cluster.
```

Comment:
s/OpenShift Container Platform/`{product-title}`/

Reply:
Updated the content.